pytorch/training/docker/1.4.0/py3/example/Dockerfile.gpu

```diff
@@ -6,6 +7,6 @@ FROM $BASE_IMAGE
 RUN pip install tensorboardX==1.6.0
 
 # Add any script or repo as required
-RUN cd /var && wget https://raw.githubusercontent.com/kubeflow/pytorch-operator/master/examples/mnist/mnist.py
```

Comment:
Please check that this change works or not.
I do not think it would work.

Reply:
testing. Not sure about the context we did on PT 1.5 about example image

Reply:
Nvm, it will work.